pages/content/posts/tpac-sevilla.md

```diff
@@ -0,0 +1,44 @@
+
+It was my first in-person TPAC event and enjoyed meeting a lot of folks that I only interacted with online so far. It was a great feeling when people recognized me or Open Web Docs and had nice things to say.
+I received kudos for Open Web Docs' setup as a [non-profit Open Collective](https://opencollective.com/open-web-docs) and I was also told that folks enjoy reading about the projects we do on MDN Web Docs. Some even watch our [GitHub project repository](https://github.com/openwebdocs/project) to hear all about our new projects immediately (cheers, Thomas Steiner!).
+
+If you attend an in-person event these days, it really is about the people. So, let me give a shout out to "my" people at TPAC: I connected with a few OWD governing board members, including Rachel Andrew (Google), Daniel Appelquist (Synk), Patrick Brosset (Microsoft), Dominique Hazael-Massieux (W3C), and Robert Nyman (Google). I also met members of OWD's steering committee, including Ruth John (Mozilla), Brian Kardell (Igalia), Lola Odelola, Brian Smith (Mozilla), and Michael[tm] Smith (W3C).
+Since our founding, we only had a single OWD meetup ([in London, earlier this year](https://openwebdocs.org/content/posts/london-meetup/)) and so it was lovely to reconnect and talk about web platform documentation once again.
+
+## Technical writing breakout session
+
+To me, the most interesting day was Wednesday. Wednesday was the day for breakout sessions in an unconference style. In collaboration with Rachel and Patrick, I chaired a breakout session: "Getting your feature adopted: learn to work with technical writers" in which we invited the participants to discuss about how to improve technical writing and documenting web technologies. We touched on topics like finding relevant Subject Matter Experts (SMEs) and presented how typically documentation gets written and maintained. [Shawn Lawton Henry](https://www.w3.org/People/Shawn/) (W3C WAI) also let us know there is currently help needed for the [Understanding WCAG](https://www.w3.org/WAI/WCAG22/Understanding/) docs and with Sam MacBeth (DuckDuckGo) we briefly talked about challenges with Webviews that ought to be documented. Find the [full minutes of our session](https://docs.google.com/document/d/1pNHLKwPs0nZ5KCY_cV05unjnL1iHKlb7CW2CQwbGGrs/edit).
+
+One thing to say about chairing and attending TPAC sessions is that I really enjoyed how scribing was taken seriously and how it is possible to be added to the speaker queue by typing "q+" in the IRC channels. You know, as an introvert it can be quite stressful to figure out the right moment to speak up in a room full of (very damn smart) people. Thanks to this, I q+'ed myself, was acknowledged by the chair and could bring up my question or comment. If you are interested in more mechanics of this, I found a blog post on the bocoup blog that has more information: [How to scribe at TPAC](https://bocoup.com/blog/how-to-scribe-at-tpac).
```

Comment:
```suggestion
There were several things I truly enjoyed about chairing and attending TPAC sessions. I really appreciate how scribing was taken seriously. I also appreciated how to be added to the speaker queue, all one has to do is type "q+" in the IRC channels. As an introvert, it can be quite stressful to figure out the right moment to speak up in a room full of (incredibly smart) people. Thanks to this inclusive feature, I q+'ed myself, was acknowledged by the chair, and was able to ask my question or share my perspective. If you are interested in more mechanics of this, I found a blog post on the Bocoup blog: [How to scribe at TPAC](https://bocoup.com/blog/how-to-scribe-at-tpac).
```